guides/capsule-installation.md

```diff
@@ -0,0 +1,96 @@
+# Capsule Installation
+
+Having our AKS cluster up and running and our kubectl has access to the API, we can go on with the capsule installation. 
+
+A dedicated namespaced will be created for capsule's objects and the installation will be done using Helm although other methods are also available.
+
+## 1. Namespace creation
+
+```bash
+$ kubectl create namespace capsule-system
+namespace/capsule-system created
+```
+
+## 2. Add Helm Chart Repository
+
+```bash
+$ helm repo add clastix https://clastix.github.io/charts
+"clastix" has been added to your repositories
+```
+
+## 3. Installing Capsule Helm Chart
```

Comment:
I am not customizing the installation, but it would be good to have the possibility of doing it. Should I just add the link to the possible arguments?

Reply:
I would say only if using a different from default

> Should I just add the link to the possible arguments?
It seems already there "capsule helm chart options"